doc/config.md

```diff
@@ -236,6 +236,10 @@ Settings list:
 
   :warning: Option does not work in Opera@12.16.
 
+* `screenshotDelay` — allows to specify a delay before making any screenshot (default: `0`).
```

Comment:
write, please, that delay is in milliseconds

Reply:
Added.